jscomp/stdlib-412/dune

```diff
@@ -1,4 +1,4 @@
-(include_subdirs unqualified)
+(copy_files# ./stdlib_modules/*.ml{,i})
```

Comment:
I am not sure the `#` is doing anything in this case. They point to e.g.

```
# 1 "jscomp/stdlib-412/stdlib_modules/weak.ml"
```

But the file is in `_opam/lib/melange/weak.ml` (due to the further copying in `(dir melange)` rule specified in `lib/dune` so it can't find the reference.

I think we can just use `copy_files`?

Reply:
yup, fine to use `copy_files`.